x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/RestEnrichTestCase.java

```diff
@@ -188,13 +193,14 @@ public void testMatchField_ImplicitFieldsList_WithStats() throws IOException {
 
     private Map<String, Object> runEsql(RestEsqlTestCase.RequestObjectBuilder requestObject) throws IOException {
         if (mode == Mode.ASYNC) {
-            return RestEsqlTestCase.runEsqlAsync(requestObject, NO_WARNINGS);
+            return RestEsqlTestCase.runEsqlAsync(requestObject, NO_WARNINGS, NO_WARNINGS_REGEX);
```

Comment:
Similar to `RestEsqlTestCase.runEsqlSync(RequestObjectBuilder requestObject)` you can create the corresponding method for `async`, if I missed it and it does exist indeed. The `async` variant is used in at least two places like this `RestEsqlTestCase.runEsqlAsync(requestObject, NO_WARNINGS, NO_WARNINGS_REGEX);`.